docs/01-Using-Fleet/03-REST-API.md

```diff
@@ -801,25 +801,22 @@ The `host_count` parameter in the software list will always be `1` in this call,
     "policies": [
       {
         "id": 1,
-        "query_id": 2,
-        "query_name": "SomeQuery",
-        "query_description": "this is a query",
+        "name": "SomeQuery",
+        "description": "this is a query",
```

Comment:
Missing `"query": "select * from foo;",` as before this was referenced by the `query_id` but that's not here anymore.

Reply:
And similarly below.